source/user-manual/reference/tools/wazuh-control.rst

```diff
@@ -26,6 +28,10 @@ The wazuh-control script is used to start, stop, configure, check on the status
 |             | This option is not available on a local Wazuh installation.                                             |
 +-------------+---------------------------------------------------------------------------------------------------------+
 | **status**  | Determine which Wazuh processes are running.                                                            |
++-------------+---------------------------------------------------------------------------------------------------------+
+| **info**    | Prints the installation type, beside the Wazuh Version and Revision, in NAME="value" pairs              |
```

Comment:
I suggest the following sentence: (including the cases):
Prints Wazuh installation type, version, and revision in environment variables format.

Reply:
Fixed!